test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1599,23 +1936,23 @@ def tearDown(self):
                                              uid='root', gid='root',
                                              mode=0644)
                             self.du.rm(hostname=self.momA, path=fn)
-                            cmd = ["rmdir", jpath]
-                            self.logger.info("deleting jobdir %s" % cmd)
+                            cmd = 'rmdir ' + jpath
+                            self.logger.info('deleting jobdir %s' % cmd)
                             self.du.run_cmd(cmd=cmd, sudo=True)
```

Comment:
Why use du.run_cmd() instead of du.rm()?

Reply:
That's more of a question for Anamika. At first glance, we're running the command "rmdir" and not "rm".